utils/schema-utils/AddJavaTypeToSchemas.ts

```diff
@@ -0,0 +1,115 @@
+import * as fs from "fs";
+import { readFile } from "fs/promises";
+
+import yargs, { Arguments } from "yargs";
+import { hideBin } from "yargs/helpers";
+
+import { getSchemaFilepaths } from "./Loaders.js";
+import { basenameRelativePathToSchemaDir } from "./PathTools.js";
+
+/**
+ * Given a schema with a known local schema_path, add a 'javaType' property for use by jsonschema2pojo
+ * @param schema_path -> Local path where schema_obj was loaded from. WILL BE OVERWRITTEN with serialized data from updated schema_obj.
+ * @param schema_inst -> OCF schema JSON to update.
+ * @param verbose -> Boolean: Display detailed logs.
+ */
+export function addJavaTypePropertyToSchema(
+  schema_path: string,
+  schema_inst: Record<string, any>,
+  verbose: boolean = false
+) {
+  let java_type_contents = `com.opencaptablecoalition.ocf.schema.${basenameRelativePathToSchemaDir(
```

Comment:
I've added `opencaptablecoalition` as domain here, thanks for the pointer @pjohnmeyer